minos/api_gateway/rest/handler.py

```diff
@@ -98,3 +121,33 @@ async def _clone_response(response: ClientResponse) -> web.Response:
     return web.Response(
         body=await response.read(), status=response.status, reason=response.reason, headers=response.headers,
     )
+
+
+async def authenticate(
+    address: str, port: str, method: str, path: str, authorization_headers: CIMultiDict
```

Comment:
I think in this case we can simply set the `authorization_headers` typing simply as `dict[str, str]` as we don't need to know that the implementation is a multidict here.

Reply:
Done.